### PR TITLE
Feature/apply more streamgages changes

### PR DESCRIPTION
**Pull request by courtneymyers**
## Related Issues:
* https://jira.epa.gov/browse/HMW-4

## Main Changes:
Latest round of streamgage edits, based on client feedback:
* Updates 'multiple measurements' text and slightly darkens the additional text shown below
* Simplifies 'show more/less categories' button functionality by combining primary and secondary parameters, and only showing the show more/less button when there are more than ten combined parameters
* Moves the 'no data available' message into the latest measurements table

## Steps To Test:
1. Navigate to http://localhost:3000/community/010900010702/overview
2. Switch to the "Monitoring & Sensors" subtab, and toggle on the "Current Water Conditions" switch
3. Expand the first streamgage ("**CAMBRIDGE RES., MET. STATION, NEAR LEXINGTON, MA**") and notice now the primary and secondary parameters are all displayed (this streamgage has 4 parameters). You can easily tell them apart, as the primary parameters have links to the glossary vs the secondary parameters that don't.
4. Expand the fifth streamgage ("**HOBBS BK BELOW CAMBRIDGE RES NR KENDALL GREEN, MA**") and notice the updated text for the water temperature parameter says "multiple measurements found". Also notice the text below has been darkened a bit (that's all "additional" text below – not just the note to see the more info link, but I think this is a good universal change, as the text was pretty light before)
5. Expand the last streamgage ("**Upper Hobbs Brook Reservoir Buoy 1**") and notice the "No data available." text has moved into the latest measurements table.

## TODO
- [x] Now that the "show more/less categories" button has been updated to only be displayed when there are more than 10 parameters/categories for a streamgage, it's quite a bit harder to find a case where that occurs. I'm sure they exist, so I'll see if I can find one... Once found, you should see the "show more/less categories" button after the first 10 parameters are displayed, and clicking it will toggle any additional parameters that exist after the initial 10.
  - To see one with more than ten go to http://localhost:3000/community/180500010403/overview and see the "SUISUN BAY A VAN SICKLE ISLAND NR PITTSBURG CA" item (last in list).

### CODE REVIEW

File: app/client/src/components/shared/WaterbodyInfo/index.js [hunk 1]
```diff
@@ -1298,65 +1296,57 @@ function UsgsStreamgagesContent({ feature }: { feature: Object }) {
...
-              <th>Latest Measurement</th>
+              <td>
+                <em>No data available.</em>
+              </td>
+              <td>&nbsp;</td>
             </tr>
-          </thead>
-          <tbody>
-            {sortedPrimaryMeasurements}
+          ) : sortedMeasurements.length < 10 ? (
```
Comment (cschwinderg) [inline app/client/src/components/shared/WaterbodyInfo/index.js:1314]:
```suggestion
          ) : sortedMeasurements.length <= 10 ? (
```

This was causing the show more/less button to be visible but not have any data under it. See "HOBBS BK BELOW CAMBRIDGE RES NR KENDALL GREEN, MA", in the example HUC you provided.  This should be a less than equal to check.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good! I did notice one issue where the show less/more button was visible but there was nothing under it (see below).

File: app/client/src/components/shared/WaterbodyInfo/index.js (hunk 1 above)
Comment (courtneymyers) [reply app/client/src/components/shared/WaterbodyInfo/index.js:1314]: Thanks, good catch!

**Review by cschwinderg — APPROVED** [review]
Thanks for making that change! Feel free to merge in.

**Pull request merged**